pydm/widgets/multi_axis_plot.py

```diff
@@ -546,6 +548,7 @@ def handleWheelEvent(self, view, ev, axis):
         for stackedView in self.stackedViews:
             if stackedView is not view:
                 stackedView.wheelEvent(ev, axis, fromSignal=True)
+        self.sigXRangeChangedManually.emit()
```

Comment:
might be helpful to add comment,
saying how we don't need check which 'axis' because only called when x-axis is changed

Reply:
Added these comments